### DISCUSSION TITLE
项目部署到OSS中

### DISCUSSION THREAD

**Discussion by sitonlotus** [jd-opensource/micro-app#729] [General]
项目如果通过OSS部署的话，如果基座使用history的话，只要一访问就会跑到文件夹下，没有对应到index.html中，在这种情况下需要怎么处理？

目前的解决方案主要有两种：
1.加CDN，但是公司里面的没人会搞，而且加了也是没什么作用；

2.给基座配置hash路由，但是子应用加载失败，希望能够给出相关的例子，目前还缺少基座和子应用都是hash的例子，

求基座和子应用都是hash的例子

**Comment by bailicangdu** [comment]
history是服务器配置的问题，将所有路由指向index.html

**Comment by bailicangdu** [comment]
hash路由的适配需要更新到1.0，使用虚拟路由，更方便接入

**Reply by sitonlotus** [reply]
你好，我这边已经按照文档升级了，也在start里面加了相关配置，但是目前基座和子应用都是使用hash目前没成功，不知道有没有相关的例子，这样好懂一点

**Reply by bailicangdu** [reply]
不要关闭虚拟路由，这样
```js
microApp.start({
  'disable-patch-request': true, // 关闭对子应用请求的拦截
})
```
